doc_source/private-clusters.md

```diff
@@ -0,0 +1,80 @@
+# Private clusters<a name="private-clusters"></a>
+
+This chapter discusses running Amazon EKS in a private cluster without outbound internet access, commonly referred to as "air-gapped".
+
+## Private cluster considerations<a name="private-cluster-considerations"></a>
+
+Here's some things to consider when running Amazon EKS in a private cluster without outbound internet access\.
++ Any container image that is not in Amazon ECR or a registry inside the VPC will need to be copied to Amazon ECR or a registry inside the VPC in order to be pulled\. See [Creating local copies of container images](#creating-local-copies-of-container-images)\.
++ **Endpoint private access** is required in order for the worker nodes to register with the cluster endpoint\. **Endpoint public access** is optional\.
++ You must bypass the EKS cluster introspection by providing the cluster certificate authority and cluster API endpoint to the worker nodes\.
++ You must include the VPC endpoints found at [VPC endpoints for private clusters](#vpc-endpoints-for-private-clusters)\.
```

Comment:
To launch fargate pods on offline private subnets, set up endpoints for ecr, s3 and sts.